packages/fxa-settings/src/models/Account.ts

```diff
@@ -764,6 +764,10 @@ export class Account implements AccountData {
         variables: { input: { state } },
       })
     );
+    const legacyLocalStorageAccount = currentAccount()!;
+    legacyLocalStorageAccount.metricsEnabled = state === 'in';
+    currentAccount(legacyLocalStorageAccount);
+    firefox.profileChanged({ metricsEnabled: this.metricsEnabled });
```

Comment:
heh, never mind my above comment about the Profile type being passed for `profileChanged`. 😛